website/templates/contact.html

```diff
@@ -4,21 +4,21 @@ <h2>Contact</h2>
         Praesent commodo cursus magna, vel scelerisque nisl consectetur. Fusce dapibus, tellus ac cursus
         commodo.</p>
 </div>
-    <div class="row bg-white">
-        <form class="bg-secondary">
+    <div class="row bg-white d-flex justify-content-center">
+        <form class="card bg-secondary p-3" style="width: 550px; height: 250px" >
             <div class="d-flex justify-content-center">
-              <label class="col-2 m-1" for="email" ><strong>Your e-mail</strong></label>
-              <input class="col-4 m-1" type="email" class="form-control" id="email" placeholder="Your email" required>
+              <label class="col-3 m-1" for="email" ><strong>Your e-mail</strong></label>
+              <input class="col-9 m-1" type="email" class="form-control" id="email" placeholder=" Your email" required>
             </div>
             <div class="form-group d-flex justify-content-center">
-                <label class="col-2 m-1" for="message"><strong>Your message</strong></label>
-                <textarea class="col-4 m-1" id="message" rows="2"
-                    placeholder="Please enter your message here..." required></textarea>
+                <label class="col-3 m-1" for="message"><strong>Your message</strong></label>
+                <textarea class="col-9 m-1" id="message" rows="4"
+                    placeholder=" Please enter your message here..." required></textarea>
             </div>
-            <div class="d-flex justify-content-end m-1">
+            <div class="d-flex justify-content-end m-2">
                 <button type="submit" class="btn btn-primary">Submit</button>
             </div>
         </div>
         </form>
-    </div>
+    </div>
 </div>
```

Comment:
The ending of the file has some issues, the last two `div` tags are unnecessary and you should probably swap the `form` in line 22 with the `div` in line 21